packages/compile-common/src/profiler/updated.ts

```diff
@@ -33,28 +33,21 @@ interface SourceFilesArtifactsUpdatedTimes {
   [filePath: string]: number; // ms since epoch
 }
 
+const getKeyFromPath = (fsPath: string): string => fsPath.split(path.sep).join("/");
```

Comment:
Oh, heh, this is a nice way of doing the replace!  I should use that elsewhere...

Reply:
:)